.scripts/util.sh

```diff
@@ -59,25 +63,66 @@ is_generated() {
     fi
 }
 
+select_browser_caller() {
```

Comment:
We MUST set the default setting at the top of this function if no option value is provided:
```bash
if [ -z $1 ]; then 
    BROWSER="chromium"
    echo "Using the default browser 'chromium'"
fi
```

Now we can pick up on the default in the next conditional block and recurse this function from the bottom if needed to keep things DRY.

Reply:
On second thought, what happens if the user has previously set a valid browser option and wants to change it? It would be better here to do:
```bash
if [ -z $1 ]; then 
    BROWSER="chromium"
    echo "Using the default browser 'chromium'"
else
   BROWSER="$1"
fi
```